roles/openstack/nova/tasks/cleanup_nodes.yml

```diff
@@ -0,0 +1,34 @@
+---
+# This file includes tasks for instances and floating IPs cleanup.
+
+# Node type can be: controller, compute, ceph, etc.
+- name: Set fact for node type
+  set_fact:
+      node_type: "{{ item }}"
+
+# These tasks done for each type of node according to 'amount' variable
+- name: Cleanup instances
+  os_server:
+      cloud: "{{ provisioner.cloud | default(omit) }}"
+      name: "{{ prefix }}{{ node_type.value.name }}-{{ item }}"
+      state: absent
+  with_sequence: "count={{ node_type.value.amount }}"
+  when: provisioner.nodes is defined
+  register: instances
+  ignore_errors: yes
+
+- name: Cleanup floating IPs
```

Comment:
This task is redundant until we change the module to delete FLIPs instead of just releasing them.